tests/test_gallery_api.py

```
@@ -576,6 +576,28 @@ def test_download_whole_gallery_as_zip(self, authenticated_client: TestClient, g
+    def test_download_whole_gallery_as_zip_accepts_mobile_get_with_bearer(self, authenticated_client: TestClient, gallery_id_fixture: str):
+        upload_photo_via_presigned(authenticated_client, gallery_id_fixture, b"mobile-bytes", "mobile.jpg")
+
+        fake_bucket = "test-bucket"
+
+        with patch("viewport.api.gallery.get_s3_settings") as mock_get_settings, patch("viewport.api.gallery.get_sync_s3_client") as mock_get_s3:
+            mock_settings = MagicMock()
+            mock_settings.bucket = fake_bucket
+            mock_get_settings.return_value = mock_settings
+
+            mock_client = MagicMock()
+            mock_client.get_object.side_effect = lambda Bucket, Key: {"Body": io.BytesIO(f"payload-{Key}".encode())}
```

Ruff (0.15.12)

[warning] Unused lambda argument: `Bucket`

(ARG005)